config/settings.yml

```diff
@@ -31,4 +31,7 @@ plugins:
     default: false
   review_tl1_users_first_post:
     default: true
+  check_spam_period_mins:
```

Comment:
`spam_check_interval_mins` would maybe be a more fitting name for this setting?

Reply:
Yeah.  I agree, interval reads better.